src/core_modules/capture-core/components/WidgetStagesAndEvents/Stages/Stage/Stage.component.js

```diff
@@ -59,31 +65,23 @@ export const StagePlain = ({ stage, events, classes, className, onCreateNew, rul
                     repeatable={repeatable}
                     enableUserAssignment={enableUserAssignment}
                     onCreateNew={onCreateNew}
-                    hiddenProgramStage={hiddenProgramStage}
+                    hiddenProgramStage={preventAddingNewEvents}
                     {...passOnProps}
                 /> : (
-                    <ConditionalTooltip
-                        content={i18n.t("You can't add any more {{ programStageName }} events", {
-                            programStageName: name,
+                    <Button
+                        small
+                        secondary
+                        disabled={preventAddingNewEvents}
```

Comment:
It seems that there are no longer any use cases where this button can be disabled. If `ConditionalTooltip` has been removed, does it make sense to remove `disabled={hiddenProgramStage}` as well?